tests/test_openmpi_settings.py

```diff
@@ -8,143 +8,49 @@
 
 import pytest
 
```

Comment:
This file looks nearly identical to `tests/test_mpi_settings.py`. Do we need them both?